app/client/views/login/loginCtrl.js

```diff
@@ -46,13 +46,23 @@ angular.module('reg')
 
       $scope.sendResetEmail = function() {
         var email = $scope.email;
-        AuthService.sendResetEmail(email);
-        sweetAlert({
+        var dialogue = {
           title: "Don't Sweat!",
           text: "An email should be sent to you shortly.",
           type: "success",
           confirmButtonColor: "#e76482"
-        });
+        };
+        if (!re.test(email)) {
+          dialogue = {
+            title: "Oops",
```

Comment:
please convert the double quotes to use single quotes instead, including the existing code :)